src/Control/FunFlow/ContentHashable.hs

```diff
@@ -65,50 +97,171 @@ class ContentHashable a where
   contentHash x = hashFinalize <$> contentHashUpdate hashInit x
 
 
-contentHashUpdate_byteArrayAccess :: ByteArrayAccess a => Context SHA256 -> View a -> IO (Context SHA256)
-contentHashUpdate_byteArrayAccess ctx a = pure $ hashUpdate ctx a
-
-contentHashUpdate_view :: ByteArrayAccess a => Context SHA256 -> View a -> IO (Context SHA256)
-contentHashUpdate_view ctx a = pure $ hashUpdate ctx a
-
 -- | Update hash context based on binary in memory representation due to 'Foreign.Storable.Storable'.
 --
 -- XXX: Do we need to worry about endianness?
 contentHashUpdate_storable :: Storable a => Context SHA256 -> a -> IO (Context SHA256)
 contentHashUpdate_storable ctx a = with a (\p -> pure $! hashUpdate ctx (MemView (castPtr p) (sizeOf a)))
 
+-- | Update hash context based on a types' 'GHC.Fingerprint.Type.Fingerprint'.
+--
+-- XXX: Is 'GHC.Fingerprint.Type.Fingerprint' sufficiently stable to be included in the content hash?
```

Comment:
Have you now clarified this, or is it still a question? If the latter, would be good to answer it sooner rather than later.

Reply:
Just checked with Alexander:
"library-name, library-version, module-name, name + the same for all field names and types"

So this should be fine. Let's remove this comment (and maybe add the fingerprint detail there)

Reply:
Thanks, I've added the info.